content/submissions.md

```diff
@@ -0,0 +1,27 @@
++++
+title = "Submissions"
+description = "Submissions"
++++
+
+## Important dates
+
+* Jan 22, 2021: Call for abstracts opens
+* Mar 9, 2021: Abstract submission closes
+* April 16, 2021: Notification of decision
+
+## Submission types
+
+### Short talk
+Focused on a package, an application note, or the presentation of a new research project that seeks input and collaborations from the community.
```

Comment:
Maybe, "or a presentation of current research. Presentations seeking collaboration or input from the community are encouraged." Only making this suggestion not to eliminate recently completed research?

Reply:
Yes, good point